config/locales/en/support_user_mailer.yml

```
@@ -5,13 +5,35 @@ en:
       body: |
         Dear %{user_name},
 
-        You have been invited to join %{service_name}.
+        You have been invited to join the %{service_name}.
```

Is this right? It reads very strangely.

"You have been invited to join the Claim funding for mentor training."

Maybe this should be?

```suggestion
        You have been invited to join the %{service_name} service.
```